app/views/ideas/index.html.erb

```diff
@@ -1,51 +1,127 @@
-      <th>It system</th>
-      <th>Title</th>
-      <th>Idea</th>
-      <th>Benefits</th>
-      <th>Impact</th>
-      <th>Involvement</th>
-      <th>Assigned User</th>
-      <th>Status</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @ideas.each do |idea| %>
-      <tr>
-        <td><%= idea.area_of_interest %></td>
-        <td><%= idea.business_area %></td>
-        <td><%= idea.it_system %></td>
-        <td><%= idea.title %></td>
-        <td><%= idea.idea %></td>
-        <td><%= idea.benefits %></td>
-        <td><%= idea.impact %></td>
-        <td><%= idea.involvement %></td>
-        <td><%= idea.assigned_user&.email %></td>
-        <td><%= idea.status %></td>
-        <td><%= link_to 'Show', idea %></td>
-        <td><%= link_to 'Edit', edit_idea_path(idea) %></td>
-        <td><%= link_to 'Destroy', idea, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
-<br>
-
-<%= link_to 'New Idea', new_idea_path %>
-<% if current_user.admin? %>
-  <%= link_to 'Users', users_path %>
-  <%= link_to 'My Assigned Ideas', ideas_path(view: "assigned") %>
-  <%= link_to 'Submitted Ideas', ideas_path(view: "submitted") %>
-<% end %>
 
+<div class="govuk-tabs" data-module="tabs">
+  <ul class="govuk-tabs__list">
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab govuk-tabs__tab--selected" href="#my-ideas">
+        My Ideas
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" href="#awaiting-approval">
+        Awaiting Approval
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" href="#review-date">
+        Review Date
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" href="#uncompleted-ideas">
+        All Uncompleted Ideas
+      </a>
+    </li>
+    <li class="govuk-tabs__list-item">
+      <a class="govuk-tabs__tab" href="#allocated-ideas">
+        My Allocated Ideas
+      </a>
+    </li>
+  </ul>
+  <section class="govuk-tabs__panel" id="my-ideas">
+    <h2 class="govuk-heading-l">My Ideas</h2>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">Idea ID</th>
+          <th class="govuk-table__header" scope="col">Title of Idea</th>
+          <th class="govuk-table__header" scope="col">Status</th>
+          <th class="govuk-table__header" scope="col">Last Updated</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+      <% @ideas.each do |idea| %>
+        <tr>
+          <td class="govuk-table__cell">Idea ID</td>
+          <td class="govuk-table__cell"><%= idea.title %></td>
+          <td class="govuk-table__cell"><%= idea.status %></td>
+          <td class="govuk-table__cell"><%= idea.updated_at %></td>
+          <td><%= link_to 'Show', idea %></td>
```

Comment:
Do we need all these actions on the index? We could omit these completely and show them when clicking into the idea. Alternatively, we could show them as buttons or something a bit nicer than 3 links. Any ideas?